### PR TITLE
#167192498 create divider atom

### PR DESCRIPTION
**Pull request by Sojisoyoye**
#### What does this PR do?

This PR creates divider field atom

#### Description of task to be completed?

write test for the feature
create style component for divider field
create divider field component

#### How should this be manually tested?

run npm install
run npm run styleguide on your console

#### Any background context you want to provide?

install node

#### What are the relevant pivotal tracker stories?

[167192498](https://www.pivotaltracker.com/n/projects/2351184/stories/167192498)

#### Screenshots (if appropriate)

image

### CODE REVIEW

**Review by frankly034 — APPROVED** [review]

**Review by akinyeleolat — APPROVED** [review]

File: src/components/UI/atoms/Divider/Divider.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+};
+
+Divider.Container = styled.span`
+${({
+    width,
+    display,
+    color,
+    theme,
+  }) => `
+border: 1px solid ${theme.borderColors[color]}
```
Comment (benfluleck) [inline src/components/UI/atoms/Divider/Divider.jsx:56]: I expect a height of 1px? 🤔

File: src/components/UI/atoms/Divider/Divider.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  color,
+  display,
+  children,
+}) => (
+  <Divider.Container
+    width={width}
+    display={display}
+    color={color}
+  >
+    {children}
```
Comment (benfluleck) [inline src/components/UI/atoms/Divider/Divider.jsx]: Why does it have children ?

File: src/components/UI/atoms/Divider/Divider.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ */
+const Divider = ({
+  width,
+  color,
+  display,
+  children,
+}) => (
+  <Divider.Container
+    width={width}
+    display={display}
```
Comment (benfluleck) [inline src/components/UI/atoms/Divider/Divider.jsx]: Remove this as dividers are mostly going to be block

File: src/components/UI/atoms/Divider/Divider.jsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+ * @return {component} Divider
+ */
+const Divider = ({
+  width,
+  color,
+  display,
+  children,
+}) => (
+  <Divider.Container
+    width={width}
```
Comment (benfluleck) [inline src/components/UI/atoms/Divider/Divider.jsx]: max-width would be more appropriate so it doesn't go beyond it

**Review by benfluleck — CHANGES_REQUESTED** [review]
Feedback left

File: src/components/UI/atoms/Divider/Divider.jsx (hunk 3 above)
Comment (Sojisoyoye) [reply src/components/UI/atoms/Divider/Divider.jsx]: I used span for the divider and span are inline by default so I need to set the display to block

Comment (benfluleck) [reply src/components/UI/atoms/Divider/Divider.jsx]: if this is the case then i suggest you use div then

Comment (Sojisoyoye) [reply src/components/UI/atoms/Divider/Divider.jsx]: Alright

File: src/components/UI/atoms/Divider/Divider.spec.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    expect(container.lastChild).toHaveStyle(`
+            border: 1px solid #B02091
+        `);
+  });
+
+  it('should render the Divider Component with width', () => {
+    const { container } = render(<Divider />);
+
+    expect(container.lastChild).toHaveStyle(`
+            width: md
```
Comment (benfluleck) [inline src/components/UI/atoms/Divider/Divider.spec.js]: Huh what is this I thought the prop had changed 🤔 . Please test for height as well

**Review by benfluleck — CHANGES_REQUESTED** [review]

**Review by benfluleck — APPROVED** [review]

**Pull request merged**